FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
Thanks for reverting.
I don't mind a force push or an additional commit. As pleases you.

And I'm not against making VS2015 the default. Sorry if I didn't make that clear. I'm just not _for_ it sneaking in with "move .sln to the root". If we'd like to consciously make 2015 the default, I'm happy to discuss. And probably will go along with it, even thought 2015 seems to be super sluggish for me. What's that all about? Scratch that. Let's talk about it later if at all.